app/models/competition.rb

```diff
@@ -1058,19 +1054,11 @@ def country_zones
     self.longitude_microdegrees = @longitude_degrees * 1e6 unless @longitude_degrees.nil?
   end
 
-  def has_events_with_ids?(event_ids)
-    (event_ids - events.ids).empty?
-  end
-
-  def has_event?(event)
-    self.events.include?(event)
-  end
```

Comment:
Again, these are fine to remove because otherwise they would conflict with the new cops